tests/unit/Backend/Synapse/AssertTest.php

```diff
@@ -320,4 +321,173 @@ public function testAssertStagingTablePass(): void
         $this->expectNotToPerformAssertions();
         Assert::assertStagingTable('#tempTableWithSharp');
     }
+
+    public function testAssertSameColumns(): void
```

Comment:
cekal bych spise provider a jeden test, ale aspon se muze pouzit `expect*`

Reply:
provider na to, že sů vstupy stejné? účel je otestovat že to správně proiteruje a porovná nechápu co tomu přidá provider. A nechápu co myslíš tím expect.

Reply:
mel jsem to napsat k tem failujicim testum, tam to dava trochu smysl, u tohoto nefailujiciho testu ne. Tam by to imho trochu zprehlednilo situaci, protoze ty testy jsou designem stejne, jen menis exception a vstupy. Ale to se uz bavime o zbytecnem leštění kulí... nech to tak

 expect jsem myslel `$this->expectExceptionMessage` apod.